nats/msgschema.go

```diff
@@ -166,12 +167,17 @@ func NewResultMsg(startedAt time.Time, result interface{}, err error) ResultMsg
 	return resultMsg
 }
 
-func EventFilter(accountId string) string {
+func EventFilter(accountId string, sourceOnly bool) string {
+	eventId := AllEventId
+	if sourceOnly == true {
+		eventId = SourceEventId
+	}
+
 	tokens := []string{
 		accountId,
 		ChannelNotify,
 		"*",
-		SourceMessageId,
+		eventId,
 	}
```

Comment:
We can fix this in a follow up, but this doesn't follow the same style as the rest where we just take the tokens as arguments rather than a bool flag. As written it's less flexible without any major benefit in readability.

Reply:
Let me fix it pre-merge. Rather do it in one go...